### PR TITLE
update cflx correctly for dms when it is obtained from the ocean

### LINKED ISSUE #39: cam_in%cflx(icol,pndx_fdms) is zero when dms is obtained from the ocean
In atm_import_export.F90, cam_in%cflx(icol,pndx_fdms) is set correctly when dms is obtained from the ocean component.
However, it is reset to 0 in chemistry.F90. So what needs to be done in the short term is to set cam_in%cflx(icol,pndx_fdms) in aero_model.F90 parallel to how it is set when dms is read from a file rather than being obtained from the ocean component.

### PR DESCRIPTION
**Pull request by mvertens**
fixes #39

### CODE REVIEW

File: src/aero_model.F90 [hunk 1]
```diff
@@ -598,11 +603,16 @@ subroutine aero_model_emissions( state, cam_in )
...
-    ! If DMS is sent from the ocean then cflx is updated in the nuopc cap
     if (.not. dms_from_ocn) then
        call oslo_aero_dms_emis(state%ncol, state%lchnk, &
             state%u(:,pver), state%v(:,pver), state%zm(:,pver), &
             cam_in%ocnfrac, cam_in%icefrac, cam_in%sst, cam_in%cflx)
+    else
+       call cnst_get_ind('DMS', pndx_fdms, abort=.true.)
+       do icol = 1,state%ncol
+          cam_in%cflx(icol,pndx_fdms) = cam_in%fdms(icol)
+          call outfld('odms', cam_in%fdms(:state%ncol), state%ncol, state%lchnk)
```
Comment (gold2718) [inline src/aero_model.F90:617]: Can you document that the `addfld` for this is in oslo_aero_ocean.F90

**Review by gold2718 — APPROVED** [review]
Please add at least one comment about what is going on here

File: src/aero_model.F90 (hunk 1 above)
Comment (mvertens) [reply src/aero_model.F90:617]: Done.

**Review by gold2718 — APPROVED** [review]
Looks good now, thanks!

**Pull request merged**